release_notes/368-release-notes.adoc

```diff
@@ -0,0 +1,137 @@
+
+[id="known-issue"]
+== Known issue
+When the same software package is installed in multiple layers in a docker image {product-title-short} may incorrectly identify the state of that package.
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-8709*: Previously, searching for CVE's with a specific severity did not returned any results. This issue has been fixed.
+* *ROX-8983*: Previously, when configuring the *Manage Watches* feature, if you added more than 12 images to the watch list, the image list would not display properly. This issue has been fixed.
+* *ROX-8276*: Previously, when the {product-title-short} Operator accessed the `central-htpasswd` secret, it would create a false positive policy violation for the *OpenShift: Advanced Cluster Security Central Admin Secret Accessed* default policy. This issue has been fixed.
+
+[id="security-update"]
+== Security update
+In earlier versions of {product-title}, the `write` permission for the `APIToken` resource allowed users to create API tokens for any role, including the `admin` role. This issue has been fixed.
+
+[id="important-system-changes"]
+== Important system changes
+
+* {product-title-short} 3.68 includes updates for the `Log4Shell` vulnerability detection policy. With this update this policy also detects CVE-2021-45046 and it includes the updated remediation based on the latest guidance by the Apache Logging security team.
+* Before this release, CVEs could be snoozed using global write access on the `Images` resource. Beginning with {product-title-short} 3.68:
+** To snooze CVEs, you must have `write` permission for the `VulnerabilityManagementRequests` resource.
+** To approve requests, you must have `write` permission for the `VulnerabilityManagementApprovals` resource.
+* When you upgrade to {product-title-short} 3.68, roles that include `write` access on the `Images` resource will have `write` permissions for both  `VulnerabilityManagementRequests` and `VulnerabilityManagementApprovals` resource.
+Red Hat recommends updating the roles to only include the least amount of resources required for each role.
+* If you have installed {product-title} using Helm, this update disabled the cluster configuration options in the {product-title-short} portal. You can continue to use Helm configuration files.
+* {product-title-short} 3.68 sends notifications for every runtime policy violation rather than sending notifications only the first encountered violation. This is the default behavior. To change it, you must set the `NOTIFY_EVERY_RUNTIME_EVENT` to `false`.
++
+[IMPORTANT]
+====
+Red Hat will remove this environment variable in future releases. Contact the support team for any related inquiries.
+====
+* Red Hat has moved the following images to new repositories:
++
+[%header,cols="1,2,2"]
+|===
+|Image
+|Old repository
+|New repository
+
+|main
```

Comment:
Please double-check the [markup chart](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). The way I read it, all the repo names and the image names should be inclosed in backticks.